web/controllers/wanikani.js

```diff
@@ -0,0 +1,60 @@
+
+const express = require('express')
+const router = express.Router();
+const wk = require('../../services/wanikani.js')
+
+// TODO: refactor this out! /////////////////////
+const db = require("../../database.js")
+function dateString(d) {
```

Comment:
`/utils` module?
i'm normally against that because something like `/utils` can quickly turn into a junk drawer, but for a small app like this i'd say it makes sense